syntax = "proto3";

message NumberDetection {
    string number = 1; //e.g "1" - "9"
}